src/main/java/traincraft/blocks/assemblytables/AssemblyTableRecipeFactory.java

```diff
@@ -0,0 +1,143 @@
+                        //find all ItemStacks for ore
+                        NonNullList<ItemStack> itemStacksNNlist = OreDictionary.getOres(jsonElement.getAsJsonObject().get("ore").getAsString());
+                        ItemStack[] itemStacks = new ItemStack[itemStacksNNlist.size()];
+                        for(int i = 0; i < itemStacksNNlist.size(); ++i){
+                            itemStacks[i] = itemStacksNNlist.get(i);
+                        }
+                        NumberedIngredient numedIng = new NumberedIngredient();
+                        numedIng.ingredient = Ingredient.fromStacks(itemStacks);
+                        if (jsonElement.getAsJsonObject().has("count")) {
+                            numedIng.setCount(jsonElement.getAsJsonObject().get("count").getAsInt());
+                        }
+                        return numedIng;
+                    } else{
+                        throw new JsonSyntaxException("Does not contain ore item.");
+                    }
+                } else{
+                    throw new JsonSyntaxException("Does not support non forge:ore_dict types.");
+                }
+            } else if(jsonElement.isJsonObject()){
+                NumberedIngredient numedIng = new NumberedIngredient();
+                numedIng.ingredient = Ingredient.fromStacks(deserializeItem(jsonElement.getAsJsonObject(), false));
+                if (jsonElement.getAsJsonObject().has("count")) {
+                    numedIng.setCount(jsonElement.getAsJsonObject().get("count").getAsInt());
+                }
+                return numedIng;
+            } else if(!jsonElement.isJsonArray()){
+                throw new JsonSyntaxException("Expected item to be object or array of objects");
+            } else{
+                JsonArray jsonarray = jsonElement.getAsJsonArray();
+                
+                if(jsonarray.size() == 0){
+                    throw new JsonSyntaxException("Item array cannot be empty, at least one item must be defined");
+                } else{
+                    ItemStack[] aitemstack = new ItemStack[jsonarray.size()];
+                    
+                    for(int i = 0; i < jsonarray.size(); ++i){
+                        aitemstack[i] = deserializeItem(JsonUtils.getJsonObject(jsonarray.get(i), "item"), false);
+                    }
+                    NumberedIngredient numedIng = new NumberedIngredient();
+                    numedIng.ingredient = Ingredient.fromStacks(aitemstack);
+                    if (jsonElement.getAsJsonObject().has("count")) {
+                        numedIng.setCount(jsonElement.getAsJsonObject().get("count").getAsInt());
+                    }
+                    return numedIng;
+                }
+            }
+        } else{
+            NumberedIngredient numedIng = new NumberedIngredient();
+            numedIng.ingredient = Ingredient.fromStacks(ItemStack.EMPTY);
+            return numedIng;
+        }
+    }
+}
```

Comment:
This can be removed in a future versions, since minecraft itself implements a "tag" system which completely replaces the ore dictionary

Reply:
in a future version